src/entities/async-exports/async-export.ts

```diff
@@ -0,0 +1,259 @@
+
+  public id?: AsyncExportPayload['id']
+  public createdAt?: AsyncExportPayload['createdAt']
+  public updatedAt?: AsyncExportPayload['updatedAt']
+  public deleted?: AsyncExportPayload['deleted']
+  public active?: AsyncExportPayload['active']
+
+  public jobReferenceId?: AsyncExportPayload['jobReferenceId']
+  public fileId?: AsyncExportPayload['fileId']
+  public signedAt?: AsyncExportPayload['signedAt']
+  public expiresAt?: AsyncExportPayload['expiresAt']
+
+  public type?: AsyncExportPayload['type']
+  public request?: AsyncExportPayload['request']
+  public staffIds?: AsyncExportPayload['staffIds']
+
+  constructor (options: AsyncExportOptions) {
+    super()
+    this.universe = options.universe
+    this.apiCarrier = options.universe
+    this.endpoint = 'api/v1/exports'
+    this.http = options.http
+    this.options = options
+    this.initialized = options.initialized ?? false
+
+    if (options?.rawPayload) {
+      this.deserialize(options.rawPayload)
+    }
+  }
+
+  protected deserialize (rawPayload: AsyncExportRawPayload): this {
+    this.setRawPayload(rawPayload)
+
+    this.type = rawPayload.type
+    this.request = rawPayload.request
+    this.staffIds = rawPayload.staff_ids
+
+    return this
+  }
+
+  public static create (payload: AsyncExportRawPayload, universe: Universe, http: Universe['http']): AsyncExport {
+    return new AsyncExport({ rawPayload: payload, universe, http, initialized: true })
+  }
+
+  public serialize (): AsyncExportRawPayload {
+    return {
+      id: this.id,
+      created_at: this.createdAt ? this.createdAt.toISOString() : undefined,
+      updated_at: this.updatedAt ? this.updatedAt.toISOString() : undefined,
+      deleted: this.deleted,
+      active: this.active,
+
+      job_reference_id: this.jobReferenceId,
+      file_id: this.fileId,
+      signed_at: this.signedAt,
+      expires_at: this.expiresAt,
+
+      type: this.type,
+      // What will this look like
+      request: this.request,
+      staff_ids: this.staffIds
+    }
+  }
+
+  public async init (): Promise<AsyncExport | undefined> {
+    try {
+      await this.fetch()
+
+      return this
+    } catch (err) {
+      throw this.handleError(new AsyncExportInitializationError(undefined, { error: err }))
+    }
+  }
+
+  // This schedules a job for exporting data. The job will be executed asynchronously.
+  // We only care about the status code here, so we don't need to return the payload.
+  public async createExport (payload: AsyncExportRawPayload | AsyncExportPayload): Promise<AxiosResponse> {
+    try {
+      const opts = {
+        method: 'POST',
+        url: `${this.universe.universeBase}/${this.endpoint}`,
+        headers: {
+          'Content-Type': 'application/json; charset=utf-8'
+        },
+        data: payload,
+        responseType: 'json'
+      }
+
+      const response = await this.http.getClient()(opts)
+
+      // Maybe not needed but for now
+      // const responseData = response.data.data[0]
```

Comment:
Question: Should this be just `data.data`? Since we only deal with a single entity at a time.

Reply:
AFAICT from the BE code I think we alwayys return an array, so the first object should be fine - I will do some typing for this though to make sure that it returns either `ayncExport | asyncExport[]` with some status codes and messages to be consumed